ext/standard/tests/time/bug38524.phpt

```diff
@@ -1,7 +1,5 @@
 --TEST--
 Bug #38524 (strptime() does not initialize the internal date storage structure)
---SKIPIF--
-<?php if (!function_exists('strptime')) echo "SKIP"; ?>
```

Comment:
This function is also only available conditionally: https://github.com/php/php-src/blob/master/ext/standard/basic_functions.c#L2733